clang/lib/CConv/ConstraintBuilder.cpp

```diff
@@ -71,13 +70,11 @@ class FunctionVisitor : public RecursiveASTVisitor<FunctionVisitor> {
       }
       if (VarDecl *VD = dyn_cast<VarDecl>(D)) {
         if (VD->isLocalVarDecl()) {
-          Info.getABoundsInfo().insertVariable(VD);
           FullSourceLoc FL = Context->getFullLoc(VD->getBeginLoc());
           SourceRange SR = VD->getSourceRange();
           if (SR.isValid() && FL.isValid() &&
               (VD->getType()->isPointerType() ||
                VD->getType()->isArrayType())) {
-            Info.addVariable(VD, Context);
```

Comment:
Same question as above --- not sure why these are being removed, since they seem separate from the change you are making?